PlayingWithMaps/ItemInfoView.swift

```diff
@@ -24,10 +27,12 @@ struct ItemInfoView: View {
     }
     
     func getLookAroundScene(){
-        lookAroundScene = nil
-        Task {
-            let request = MKLookAroundSceneRequest(mapItem: selectedResult)
-            lookAroundScene = try? await request.scene
+        if let selectedResult = selectedResult {
+            lookAroundScene = nil
+            Task {
+                let request = MKLookAroundSceneRequest(mapItem: selectedResult)
+                lookAroundScene = try? await request.scene
+            }
```

Comment:
I didn't really look into this method, but given that it is a potentially nil object, I wrapped it.